integration/sawtooth_integration/tests/test_py_executor.rs

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+extern crate intkey_rust;
+extern crate intkey_workload;
+extern crate protobuf;
+extern crate sawtooth_perf;
+extern crate sawtooth_sdk;
+extern crate sawtooth_validator;
+
+use std::thread;
+
+use protobuf::Message;
+
+use self::intkey_rust::start;
+use self::intkey_workload::intkey_iterator::IntKeyIterator;
+use self::intkey_workload::intkey_transformer::IntKeyTransformer;
+use self::sawtooth_perf::batch_gen::SignedBatchIterator;
+use self::sawtooth_sdk::signing;
+use self::sawtooth_sdk::messages::batch::Batch;
+use self::sawtooth_validator::execution::execution_platform::{ExecutionPlatform, NULL_STATE_HASH};
+use self::sawtooth_validator::execution::py_executor::PyExecutor;
+use self::sawtooth_validator::proto::batch;
```

Comment:
I do not believe you need `self::` here

Reply:
This way allows for there not to be a src directory.